src/lustre/lustreAstHelpers.mli

```diff
@@ -77,3 +79,17 @@ val is_type_signed_machine_int: lustre_type -> bool
 val is_type_machine_int: lustre_type -> bool
 
 val is_machine_type_of_associated_width: (lustre_type * lustre_type) -> bool
+
+val is_type_or_const_decl: declaration -> bool
+
+val split_program: declaration list -> (declaration list * declaration list)
+
+val abstract_pre_subexpressions: expr -> expr
+
+val extract_node_equation: node_item -> (eq_lhs * expr) list
+
+val get_last_node_name: declaration list -> ident option
+
+val move_node_to_last: ident -> declaration list -> declaration list
```

Comment:
You should document these functions. Without context, inferring what they do is not straightforward.

Reply:
done